src/fulltrade/BarrierCard.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes, Component } from 'react';
+import BarrierInput from './BarrierInput';
+
+export default class BarrierCard extends Component {
+    static propTypes = {
+        barrier: PropTypes.number,
+        barrier2: PropTypes.number,
+        barrier1Info: PropTypes.shape({
```

Comment:
To differentiate them from the actual barrier value ~ any suggestion?

the barrierInfo describe name and default value of barrier1